MAINTAINERS

```diff
@@ -61,6 +61,13 @@ M: Pavel Roschin <rpg89(at)post(dot)ru>
 W: http://plugins.geany.org/defineformat.html
 S: Maintained
 
+devhelp
+P: Matthew Brush <matt@geany.org>
```

Comment:
Have you got Matthew's agreement to be maintainer?

Reply:
Then its rude to put him as maintainer, nominate yourself.  If you are not willing to support the plugin you are proposing it will be less likely to be accepted back.

Reply:
Can I put my name and email in place of theirs and their authorization?

Reply:
To be clear, the plugin has been removed from Geany-plugins because it was not usable as it was and broke the build. You are effectively submitting a new plugin that is now usable, not Matthew.  

He and any other authors retain copyright to the code that is reused, you can't change that, but it is licensed under the GPL allowing you to reuse it as you are retaining the GPL.  That is the authorisation.  What the maintainers file says has no impact on that. 

Preferably you should use Git to show the changes from a position when the plugin was in the repository so the copyright is traceable.  I am not a git expert so I have not considered exactly what that would entail, likely reverting the removal commits, then applying your changes on top of that.

The maintainers file is a list of contacts for the person who is willing to provide support (address issues, fix bugs, maybe add features) simply to make it easier for users to find such support.  In many cases that is not the author.  To avoid "drive by" plugin submissions they normally are not accepted unless the submitter is willing to stand behind their work (at least for a while).